scripts/qiita

```diff
@@ -392,12 +391,8 @@ def start(port, master):
             raise
     base_url = qiita_config.base_url
 
-    if search(r":\d+$", base_url):
-        tmp = "Qiita started on %s @ %s" % (base_url, ctime())
-    else:
-        tmp = "Qiita started on %s:%d @ %s" % (base_url, port, ctime())
-
-    click.echo(tmp)
+    click.echo("Qiita started on base_url: %s, port: %d @ %s" % (
+        base_url, port, ctime()))
```

Comment:
just to confirm, is there a possibility we'll have messages during startup that will say http://localhost:21174:21174 again or ?